### PR TITLE
OADP-206: BSL and VSL credentials

### PR DESCRIPTION
**Pull request by apinnick**
https://issues.redhat.com/browse/OADP-206

Credentials for backup locations and snapshot locations.

Previews:

https://deploy-preview-40688--osdocs.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/installing/about-installing-oadp.html

### CODE REVIEW

**Comment by mperetzred** [comment]
@apinnick, we need to explain the default secret name:
https://github.com/openshift/oadp-operator/blob/master/docs/credentials.md#default-secret-names
https://github.com/openshift/oadp-operator/blob/master/docs/credentials.md#credentials-for-volumesnapshotlocation
https://github.com/openshift/oadp-operator/blob/master/docs/credentials.md#separate-credentials-for-bsl-and-vsl
TL;DR: for BSL - secret name can be custom, for snapshots a secret with a **default name** must exist.
Better to quote the doc.

**Comment by apinnick** [comment]
> @apinnick, we need to explain the default secret name: https://github.com/openshift/oadp-operator/blob/master/docs/credentials.md#default-secret-names https://github.com/openshift/oadp-operator/blob/master/docs/credentials.md#credentials-for-volumesnapshotlocation https://github.com/openshift/oadp-operator/blob/master/docs/credentials.md#separate-credentials-for-bsl-and-vsl TL;DR: for BSL - secret name can be custom, for snapshots a secret with a **default name** must exist. Better to quote the doc. This doc https://github.com/openshift/oadp-operator/blob/master/docs/credentials.md is not referred there.

@mperetzred Did you see that this is a draft PR? I haven't started working on it.
